modules/charging.h

```diff
@@ -59,7 +59,7 @@
 
 /** Battery level at which to enable charging */
 # define MCE_SETTING_CHARGING_LIMIT_ENABLE      MCE_SETTING_CHARGING_PATH "/limit_enable"
-# define MCE_DEFAULT_CHARGING_LIMIT_ENABLE      (MCE_DEFAULT_CHARGING_LIMIT_DISABLE - 3)
+# define MCE_DEFAULT_CHARGING_LIMIT_ENABLE      87
```

Comment:
So this value will need to be remembered to recalc manually, if `MCE_DEFAULT_CHARGING_LIMIT_DISABLE` changes in future?
Perhaps could do with a comment of calculation:)

Reply:
The problem here is completely bogus value. Having valid values drift apart is less problematic. If such things would happen (say - hard coded values at settings ui are changed), it is just a visual glitch.

[The usual approach with things like this would be to have them defined and configurable in mce and exposed to settings ui which then proceeds to ignore all of it. Here we just skip the middle part and hardcode at  both ends.]